README.md

```diff
@@ -10,6 +10,30 @@ This plugin provides a [Nodestream](https://github.com/nodestream-proj/nodestrea
 pip install nodestream-plugin-neptune
 ```
 
+**NOTE:** This plugin is currently only available in a pre-release version. When creating a new project with
+`nodestream new test_project_name --database neptune`, you may need to update the generated pyproject.toml file to
+use the latest pre-release version of the Neptune plugin from PyPI. For example:
+
+```toml
+# ...
+[tool.poetry.dependencies]
+python = "^3.10"
+nodestream = "^0.12.0a2"
+
+nodestream-plugin-neptune = "^0.12"
+# ...
+```
+should be updated to
+```toml
+# ...
+[tool.poetry.dependencies]
+python = "^3.10"
+nodestream = "^0.12.0a2"
+
+nodestream-plugin-neptune = "^0.12.0a1"
```

Comment:
I think we're going to need to merge make some new releases since we've had some changes in both the core nodestream (bug fixes) and the neptune connector.

```suggestion
nodestream = "^0.12.0a3"
nodestream-plugin-neptune = "^0.12.0a2"
```